src/pages/waiting-room/waiting-room.html

```diff
@@ -81,23 +81,30 @@ <h4>{{ 'waitingRoom.declarationIntent' | translate}}:</h4>
           </ion-row>
         </ion-col>
       </ion-row>
+      
       <div class="signature-area-padding"></div>
       <ion-row>
         <div class="validation-bar" [class.ng-invalid]="isCtrlDirtyAndInvalid('signatureAreaCtrl')"></div>
         <ion-col>
           <ion-row>
             <ion-col>
-              <div id="waiting-room-signature-validation-text" class="validation-text-with-row-spacing"
+            </ion-col>
+          </ion-row>
+          <ion-row>
+            <ion-col col-3></ion-col>
+            <ion-col col-24><h5>{{ 'waitingRoom.signaturePrompt' | translate }}</h5></ion-col>
+            <ion-col>
+                <div id="waiting-room-signature-validation-text" class="validation-text-with-row-spacing"
                 [class.ng-invalid]="isCtrlDirtyAndInvalid('signatureAreaCtrl')">
                 {{ 'waitingRoom.signatureValidation' | translate }}
               </div>
             </ion-col>
-          </ion-row>
-          <ion-row class="declaration-signature-row">
-            <ion-col class="declaration-signature-col">
+            </ion-row>
+            <ion-row class="declaration-signature-row">
+              <ion-col class="declaration-signature-col">
               <signature-area formControlName="signatureAreaCtrl" [signature]="pageState.signature$ | async"
                 retryButtonText="{{ 'waitingRoom.signatureRetry' | translate }}"
-                signHereText="{{ 'waitingRoom.signaturePrompt' | translate }}">
+                signHereText=" ">
```

Comment:
Are we doing this on the health declaration page as well for consistency 

Also not happy with the signHereText being an empty string can we remove it and if it's not used in either signature box use. Remove it from the component.